src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -316,12 +343,8 @@ static public String getPublicGateway() {
     return null;
   }
 
-  static public ServiceInterface create(String name, String type) {
-    return create(null, name, type);
-  }
-
-  static public ServiceInterface create(String configName) {
-    return create(configName, null, null);
+  static public ServiceInterface create(String name) {

```

Comment:
This needs a javadoc. Also potentially problematic that it has the same signature as a previous method. What's the purpose of this overload? As far as I can tell name needs to be non-null and must be an entry in the plan, but beyond that the control flow gets very messy

Reply:
Its not an overload ..
start(name) - allows the starting of a service only based on name - requirements would be there is a service config file available.
The old create(String configName) was removed an created/started a config set.
Now its more appropriately named startConfig(configName)

Reply:
added javadoc